types/bull-arena/index.d.ts

```diff
@@ -18,12 +15,24 @@ declare function Arena(
 
 declare namespace BullArena {
     interface MiddlewareOptions {
-        Bull?: typeof Bull;
-        Bee?: typeof Bee;
-        BullMQ?: typeof Queue;
+        Bull?: QueueConstructor;
+        Bee?: QueueConstructor;
+        BullMQ?: QueueConstructor;
         queues: Array<QueueOptions & ConnectionOptions>;
     }
 
+    interface QueueConstructor {
+        new (queueName: string, opts?: QueueOptions): Queue;
+    }
+
+    interface Queue {
+        // Interface of Queue is much larger and
+        // inconsistent between different packages.
+        // We are using an example method here
+        // that is consistent across all providers.
+        getJob(jobId: string): Promise<unknown>;
```

Comment:
I have tested your changes on all the queues supported by bull-arena. Code changes are working fine. If we are planning to add one more queue option in bull-arena then that queue have to have getJob method.